### PR TITLE
Release notes for MATE 1.22

### PR DESCRIPTION
**Pull request by vkareh**

### CODE REVIEW

**Comment by yetist** [comment]
Good job.:+1:

File: blog/20190318-mate-1-22-released.md [hunk 1]
```diff
@@ -0,0 +1,2140 @@
...
+.. tags: Releases
+.. date: 2019-03-18 09:00:00
+.. title: MATE 1.22 released
+.. slug: 2019-03-18-mate-1-22-released
+.. author: Victor Kareh
+-->
+
+After about a year of development, the MATE Desktop team have finally released MATE 1.22. A big thank you to all contributors who helped to make this happen.
+
+![about|(/assets/img/blog/about-mate-1.22.png)
```
Comment (selectiveduplicate) [inline blog/20190318-mate-1-22-released.md]: The image doesn't show up here on a local build! :confused:
Shouldn't it be ``about``?

**Comment by mate-user** [comment]
I have some suggestions:
thumbnail all rarfiles --> thumbnail all **RAR files** ? (2 of these, one might be a duplicate, they both show the same "comics: fix incorrect mimetype saying to open and thumbnail all rarfiles" !)

hi-dpi --> **HiDPI** ? (4 of these)

Read authors and documenters fom caja.about file --> Read authors and documenters **from** caja.about file

GSettings should init beforce use it --> GSettings should init **before** use it ?

Under Caja:
- add option to open multiple uri in tabs at startup --> add option to open multiple **URIs** in tabs at startup
- Revert "add option to open multiple uri in tabs at startup" --> Revert "add option to open multiple **URIs** in tabs at startup"
- add option to open multiple uri in tabs at startup --> add option to open multiple **URIs** in tabs at startup

desktop-canvas-view: scale desktop workarea --> desktop-canvas-view: scale desktop **work area** ? (2 of these, maybe one is a duplicate?)
Fix url of ulinks to point to mate-user-guide --> Fix **URL** of ulinks to point to mate-user-guide
Update Catalan screenhots --> Update Catalan **screenshots** (2 of these, maybe one is a duplicate?)
pre-bumb version --> pre-**bump** version
EogScrollView: Implement simple two-pass filtering --> **Eom**ScrollView: Implement simple two-pass filtering (2 of these, one may be a duplicate?)
EogMetadataSidebar --> **Eom**MetadataSidebar or MetadataSidebar ?
TotemScrSaver --> Totem**ScreenSaver**

under libmatekbd:
- transifex: add missing optins to config file --> transifex: add missing **options** to config file (2 of these, 1 duplicate?)
- correct scaling on HiPDI displays --> correct scaling on Hi**DPI** displays (2 of these, 1 duplicate?)

under libmatemixer:
add Travic-CI support --> add **Travis**-CI support

multiload: ignore priviledged mount points --> multiload: ignore **privileged** mount points (2 of these, 1 duplicate?)
Use orderedlist --> Use **ordered list** 
timerapplet: Allow reseting --> timerapplet: Allow **resetting**
Remove dublicated assignment --> Remove **duplicated** assignment
Fix assertion failture on g_icon_to_string --> Fix assertion **failure** on g_icon_to_string
udpate examples --> **update** examples
not passing on clockbutton --> not passing on **clock button** ?
update mate-menus requiremnt --> update mate-menus **requirement**
Theme: consistant theming for applet drag handles in all themes --> Theme: **consistent** theming for applet drag handles in all themes (2 of these, 1 duplicate?)
xrandr-applet-popup: dim menuitems for disabled monitor --> xrandr-applet-popup: dim **menu items** for disabled monitor (3 of these)
xrander-applet-popup --> **xrandr**-applet-popup
drop umtp suport --> drop umtp **support**
Update Catalan sceenshots of mate-dictionary --> Update Catalan **screenshots** of mate-dictionary (2 of these, 1 duplicate?)
Don't loss the PRIMARY clipboard --> Don't **lose** the PRIMARY clipboard
hide the frist tab --> hide the **first** tab
add the abbility --> add the **ability**

There may be more duplicate entries in these release notes, hope this helps!

**Comment by raveit65** [comment]
> not passing on clockbutton --> not passing on clock button ?

`clockbutton` is just the name in code for a special button.

> xrandr-applet-popup: dim menuitems for disabled monitor --> xrandr-applet-popup: dim menu items for disabled monitor 

`menuitem` is a widget name by Gtk+

> There may be more duplicate entries in these release notes

Not sure if this are duplicate entries because they have different commit-ids.
Any way, i will delete them.

**Comment by raveit65** [comment]
https://github.com/mate-desktop/mate-desktop.org/pull/154

**Comment by raveit65** [comment]
Btw. can do just simple a pull request next time, please?
This will help a lot.

**Comment by mate-user** [comment]
How would I do a pull request ?
I copy/pasted into Google Docs to find typos, maybe that will be useful when you write the release notes for MATE 1.24 !

**Comment by mate-user** [comment]
The MATE 1.20 blog post still has some typos, see this:
https://github.com/mate-desktop/mate-desktop.org/commit/9ab6e436541e922ab08344a9742eb811f7724033#commitcomment-32725270

**Comment by raveit65** [comment]
1) Fork mate-desktop.org to you own github repo. 
2) Clone you repo to your local machine.
3) Edit https://github.com/mate-desktop/mate-desktop.org/blob/master/blog/20180206-mate-1-20-released.md
https://github.com/mate-desktop/mate-desktop.org/blob/master/blog/20180206-mate-1-20-released.zh_cn.md
with your fixes.
4) commit your changes and push them to your github repo
6) Do a pull request

That's it ;)

**Comment by mate-user** [comment]
Shame it isn't easier to suggest changes here.

**Pull request merged**